lib/modules/apostrophe-images/public/css/grid-item.less

```diff
@@ -23,8 +23,17 @@
     width: 100%;
     height: auto;
 
+    @supports(display: grid) {
+      text-align: center;
```

Comment:
I'd suggest using flex here to center the image, both horizontally and vertically. The image hugs the top now, and landscape files like wide logos look odd.

```
    display: flex;
    align-items: center;
    justify-content: center;
```

But prefixed.